src/theme/NotFound.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from 'react';
+import Layout from '@theme/Layout';
+import Head from '@docusaurus/Head';
+import BrowserOnly from '@docusaurus/BrowserOnly';
+
+
+function NotFound() {
+  return (
+    <Layout title="Page Not Found">
+      <Head>
+        <BrowserOnly>
+        <script>
+        {
+            window.onload = () => {
+              document.referrer.includes("/stacks") ? window.location = document.referrer.replace("/stacks", "") : "";
```

Comment:
Is it a good idea to have a check for `amido.github.io` and only do the redirect if the URL starts with that?

Not that we'll probably ever have stacks.amido.com/stacks lol.

Reply:
It will not start with "amido.github.io" the alias in CNAME changes the address automatically

Reply:
> If we just want `/stacks` to redirect it might be better to have a static page under `/stacks`

What would happen with all the other pages under /stacks/?

Reply:
People can still view the site under the old URL of amido.github.io/stacks, that's why we need this check in the first place?